proto/cosmos/distribution/genesis.proto

```diff
@@ -116,10 +116,10 @@ message ValidatorSlashEventRecord {
   ];
   uint64 height = 2;
   uint64 period = 3;
-  ValidatorSlashEvent event = 4 [
+  ValidatorSlashEvent validator_slash_event = 4 [
```

Comment:
reason: the json field was `validator_slash_event` in v0.39

https://github.com/cosmos/cosmos-sdk/blob/89ceb62a463e783950d559fbe7164613565edae5/x/distribution/types/genesis.go#L51